### ISSUE TITLE
Colab?

### ISSUE THREAD

**Issue by robbsaber** [MingtaoGuo/StyleSwap#1]
Has anyone tried this on colab?

**Comment by MingtaoGuo** [comment]
Sorry, the model is still training now, because of the limited computation resources and the loss weights need to be adjusted,   training iteration is 60 000, it's not enough to use, as follows, 
![](https://github.com/MingtaoGuo/StyleSwap/blob/main/IMGS/061400.png)

**Comment by MingtaoGuo** [comment]
Hi!@netrunner-exe, Thank you for your interest in this repo, I use the FFHQ dataset for training, it may be not enough, because the size of FFHQ just 70 000, I recommend you to use VGGFace2 whose data size over 2M face. For the high resolution model like 512 or 1024, the repo is supported but i didn't test it : ), if you would like to use 512, you should enhance VGGFace2 dataset to 512x512 by GPEN like in the paper.

**Comment by MingtaoGuo** [comment]
> > Hi!@netrunner-exe, Thank you for your interest in this repo, I use the FFHQ dataset for training, it may be not enough, because the size of FFHQ just 70 000, I recommend you to use VGGFace2 whose data size over 2M face. For the high resolution model like 512 or 1024, the repo is supported but i didn't test it : ), if you would like to use 512, you should enhance VGGFace2 dataset to 512x512 by GPEN like in the paper.
> 
> Thanks for the quick response! As an idea, the dataset can also be processed through CodeFormer or GFPGAN 1.4. Do you have plans to publish test code to test the pretrained model with photos and videos?

I will publish the photo and video test code after the training phase is done, please waiting : - )

**Comment by skyler14** [comment]
How long does the training take on what hardware roughly btw?

And for the model stuff is there any face specific fine-tuning phase or pretrained this works basically out the box?

**Comment by MingtaoGuo** [comment]
> How long does the training take on what hardware roughly btw?
> 
> And for the model stuff is there any face specific fine-tuning phase or pretrained this works basically out the box?

Hi, the model is still training now. I don't know why the id loss reduce so slowly :-(, the author doesn't release the training code and nobody know the training details, so I adjust the training weights all the time, oh my goodness, it's so horrible :-<.

**Comment by loboere** [comment]
Any progress when you will release the trained model?

**Comment by micjackfr** [comment]
Hello @MingtaoGuo,
I see in train_simple.py , "add_argument" 
size 256
channel_multiplier, default=2
narrow, default=1.0

For size 256 it's not 1 and 0.5 respectively ?

**Comment by tterpell** [comment]
How is it looking?
